config/core/configmaps/features.yaml

```diff
@@ -22,3 +22,5 @@ metadata:
     knative.dev/config-propagation: original
     knative.dev/config-category: eventing
 data:
+  # ALPHA feature: The kreference-group allows you to use the Group field in KReferences.
+  kreference-group: "false"
```

Comment:
+1 to @dprotaso 's comment about "false" being a bad choice here, it is not clear what 'false' means. 

>  Make use of tristate values (enable, allowed, disabled) instead of a boolean value (false,true).

Reply:
That's a leftover, that was already changed in the flags CM code, thanks for noticing it